selfdrive/locationd/laikad.py

```diff
@@ -112,18 +116,27 @@ def process_ublox_msg(self, ublox_msg, ublox_mono_time: int, block=False):
       new_meas = [m for m in new_meas if 1e7 < m.observables['C1C'] < 3e7]
 
       processed_measurements = process_measurements(new_meas, self.astro_dog)
+      self.update_pos_fix(t, processed_measurements)
+      est_pos = self.gnss_kf.x[GStates.ECEF_POS]
 
-      est_pos = self.get_est_pos(t, processed_measurements)
+      allow_incomplete_delay = any(self.gnss_kf.P[GStates.ECEF_POS].diagonal() > 1e3)
```

Comment:
Need abs?

Reply:
Before I wasnt using the diagonal and then needed abs. But I think in all cases it can be removed now.